### PR TITLE
DOC: update Code of Conduct committee

### PR DESCRIPTION
**Pull request by rgommers**
See https://mail.python.org/pipermail/numpy-discussion/2020-May/080625.html for the mailing list conversation on the new CoC committee composition.

Also updated the list of people under Institutional Partners now that I'm looking at people's names.

When this is merged I'll deal with adding people to numpy-conduct@googlegroups.com and taking Nathaniel and myself off that list.

### CODE REVIEW

**Review by anirudh2290 — APPROVED** [review]

**Review by melissawm — APPROVED** [review]

**Review by WarrenWeckesser — APPROVED** [review]

**Review by njsmith — APPROVED** [review]

**Review by charris — APPROVED** [review]

**Review by seberg — APPROVED** [review]

**Review by mattip — APPROVED** [review]

**Comment by mattip** [comment]
Of the people mentioned in the PR, @tylerjereddy (removed), @rossbar (added) and Mars Lee (added) have yet to approve. It would be nice to get their approval before merging.

**Review by rossbar — APPROVED** [review]

**Comment by rgommers** [comment]
Cc'ing @marsbarlee as well. She is away right now so may not answer soon - that's not blocking for merge here.

**Review by stefanv — APPROVED** [review]

**Comment by seberg** [comment]
Thanks everyone! Lets put this in and then transfer mailing list ownerships for the CoC list.

@anirudh2290 we may want to add your affiliation as well soon, after all its just as much about transparency, I am not sure we have too clear rules and I can understand not rushing that, but I think this list is largely about being transparent right now.

**Comment by rgommers** [comment]
> @anirudh2290 we may want to add your affiliation as well soon, after all its just as much about transparency, I am not sure we have too clear rules and I can understand not rushing that, but I think this list is largely about being transparent right now.

The CoC and Institutional Partners lists are unrelated. I have mentioned the latter to Brian, but he wasn't in a hurry with that - we'll revisit it in a couple of months. There's permissions involved, can't just grab a logo ....

**Comment by tylerjereddy** [comment]
Looks good--sorry I do struggle to keep up with all the notifications :)

**Pull request merged**
